### PR TITLE
[msal-angular-v2] Adding B2C sample

### PR DESCRIPTION
**Pull request by derisen**
This PR adds a B2C sample to **msal-angular-v2** samples.

p.s. currently the `edit_profile` flow forces the user to sign-out after edits are submitted. I'll update in another PR once I settle on a safe and straightforward pattern for selecting accounts in multi-policy settings (across all 2.x b2c samples).

### CODE REVIEW

File: samples/msal-angular-v2-samples/angular11-b2c-sample/src/app/profile/profile.component.ts [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  ) { }
+
+  ngOnInit() {
+    this.getProfile(apiConfig.uri);
+  }
+
+  getProfile(url: string) {
+    this.http.get(url)
+      .subscribe(profile => {
+        console.log(profile)
```
Comment (jo-arroyo) [inline samples/msal-angular-v2-samples/angular11-b2c-sample/src/app/profile/profile.component.ts:28]: Nit: should this be left in?

**Review by jo-arroyo — APPROVED** [review]
Looks great! This sample will probably need to be merged into the `msal-angular-v2` branch instead of `dev` though, as that is where all the angular v2 code and samples are living at the moment.

File: samples/msal-angular-v2-samples/angular11-b2c-sample/src/app/profile/profile.component.ts (hunk 1 above)
Comment (derisen) [reply samples/msal-angular-v2-samples/angular11-b2c-sample/src/app/profile/profile.component.ts:28]: hmm, looks like it's the same with other Angular samples 😕

Comment (jo-arroyo) [reply samples/msal-angular-v2-samples/angular11-b2c-sample/src/app/profile/profile.component.ts:28]: Okay, that's fine. Just checking

**Review by jasonnutter — APPROVED** [review]

**Pull request merged**